scripts/01-get-devtools-cran-table.R

```diff
@@ -26,12 +26,26 @@ define_parser <- function() {
       c("--config"), type = "character",
       help = paste(
         "A .yaml file containing the configuration details for the workflow.",
-        "The .yaml should contain fields for the keys",
-        "'task_view_url' (URL for the CRAN task-view .ctv file);",
-        "'cran_details_file' (the output of this script);",
-        "and 'drop' (which packages that are in both CRAN and the task-view",
+        "The .yaml should contain fields for the keys:",
```

Comment:
pass in comma-separated string of dropped-packages, or temp file containing the dropped-package names